### PR TITLE
Updated SBC Version and Add Disable Prop to Header

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity###

*Description of changes:*

* Updated the SBC Common Components version.

* Implemented Prop to disable/hide the Login Functionality for MVP.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "http-status-codes": "^1.4.0",
     "moment": "^2.27.0",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "2.4.5",
+    "sbc-common-components": "2.4.6",
```
Comment (severinbeauvais) [inline client/package.json:22]: Why does line 3 still show 0.1.6 ? It should show 0.2.0. And then I was going to say, please bump it up to 0.2.1 :)

File: client/src/App.vue [hunk 2]
```diff
@@ -23,6 +23,7 @@
         :redirectOnLoginSuccess="getNameRequestUrl"
         :redirect-on-login-fail="getNameRequestUrl"
         :redirect-on-logout="getNameRequestUrl"
+        :show-actions="false"
```
Comment (severinbeauvais) [inline client/src/App.vue:26]: So, technically it's an enable prop, not a disable prop 😛 

(No change required! Just ribbing ya.)

**Review by lekshmimallika-aot — APPROVED** [review]

**Review by vysakh-menon-aot — APPROVED** [review]

File: client/package.json (hunk 1 above)
Comment (cameron-freshworks) [reply client/package.json:22]: Copy that! Will do

**Review by severinbeauvais — APPROVED** [review]
GTG as soon as you figure out the version issue.

File: client/src/App.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/App.vue:26]: lol ya got me! 

It was initially a disable prop... many many moons ago 😆

File: client/package.json (hunk 1 above)
Comment (severinbeauvais) [reply client/package.json:22]: As there is no 0.2.0 yet, just change it to that.

**Comment by cameron-freshworks** [comment]
> GTG as soon as you figure out the version issue.

Updated to 0.2.1!

**Pull request merged**